src/libmongoc/src/mongoc/mongoc-apm.h

```diff
@@ -160,6 +160,9 @@ mongoc_apm_command_failed_get_duration (
 MONGOC_EXPORT (const char *)
 mongoc_apm_command_failed_get_command_name (
    const mongoc_apm_command_failed_t *event);
+MONGOC_EXPORT (const char *)
+mongoc_apm_command_failed_get_database_name (
+   const mongoc_apm_command_failed_t *event);
```

Comment:
Add doc pages for `mongoc_apm_command_failed_get_database_name` and the to-be-added `mongoc_apm_command_succeeded_get_database_name`. See `mongoc_apm_command_started_get_database_name.rst` for an example.